cloudnetwork/v1/001-cloudprivateipconfig.crd.yaml

```
@@ -29,7 +29,7 @@ spec:
                 name:
                   anyOf:
                     - format: ipv4
-                    - format: ipv6
+                    - pattern: '[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}\.[0-9a-f]{4}'
```

Looking at other `pattern`s in the API it should match line start/end, additionally I think we can simplify it a bit:
```suggestion
                    - pattern: '^[0-9a-f]{4}(\.[0-9a-f]{4}){7}$'
```